remarkable-review/The End of Paper.md

```diff
@@ -0,0 +1,101 @@
+
+If you're really into organizational systems, I recorded a [livestream](livestream) where I whiteboarded my reading, writing, and note-taking systems.
+
+My digital note repository does have advantages over my multitudinous Moleskines.
+
+**Search.** I can instantly summon all the notes I've written in the past decade containing a search term. This is tremendously helpful when I know what I'm looking for and I want to go there right away.
+
+Conversely, my paper notebooks are organized in a linear fashion, with each day's consecutive note following the previous one. I attempted to use indexes and break up sections by topic, but it never stuck. Currently, I just have tabs that allow me to quickly jump to a specific date.
+
+<picture of my paper notebooks with date tabs>
+
+**Organization.** In addition to search, digital notes can be organized in a variety of non-linear ways, including tags and folders. Tags and folders can be helpful, but surprisingly it hasn't proved as valuable as I expected. I sometimes use a specific tag to focus in on a set of notes and "hide" my other notes, but for some reason a hierarchical tagging system hasn't really helped me.
+
+The paper equivalent would be if paper pages could be removed from their binding and reshuffled.
+
+<TODO: Cut this one?>
+
+**Sharing and collaboration.** I love the ease with which I can share a digital note with coworkers or migrate them to a collaborative document like Google Docs or Notion. Paper is still hard to share. Even with ubiquitous smartphones with cameras, wifi, and apps to quickly "scan" a document (I use the excellent [Office Lens](https://apps.apple.com/us/app/microsoft-office-lens-pdf-scan/id975925059)), it's still not as easy to share paper as it is digital text.
+
+**Digital workflow integration.** Digital text is incredibly flexible. I'm writing this article in Bear. When I'm finished, I'll move the text to my [blog](https://github.com/jessmartin/jessmartdotin-gatsby), add some frontmatter at the top, check it into source control, and the article will instantly appear as an HTML page on my website. That magical experience is made possible by a structured text format called Markdown. This is just one example of integration into a digital workflow. Text can be inserted, extracted, transformed, all via simple scripts. That's simply not possible with paper.
+
+**Archival.** If my house were on fire, the first (non-living) thing I would grab would be my notebooks. I once lost a half-filled notebook, a series of notes I was taking while living in Ireland. It's still a little painful to think about. I never worry about losing my digital notes.
+
+**Iteration.** Digital notes can be modified. It's just text! I frequently revisit, extend, adapt, and modify my digital notes. It's something I'm trying to get better at. But once something is laid to paper, it can't really be changed.
+
+## Why the reMarkable has stolen my heart
+I say "holy smokes" way too often.
```

Comment:
I _love_ how up until now you were storytelling. Keep doing that here! Perhaps start by saying something like "Despite how amazing digital notes can be, they don't facilitate "x" the same way. Or at least that's what I thought..." Cue dramatic music, etc.